pyro/distributions/zero_inflated.py

```diff
@@ -11,7 +11,8 @@
 )
 from torch.nn.functional import softplus
 
-from pyro.distributions import NegativeBinomial, Poisson, TorchDistribution
+from torch.distributions import NegativeBinomial, Poisson
+from pyro.distributions import TorchDistribution
```

Comment:
Can you explain more about your motivation here?  Was something breaking?

My concern is that the pyro imports are actually [wrappers](https://github.com/pyro-ppl/pyro/blob/727aff741e105715840bfdafee5bfeda7e8b65e8/pyro/distributions/torch.py#L350) around the torch versions that add a [mixin class](https://github.com/pyro-ppl/pyro/blob/dev/pyro/distributions/torch_distribution.py#L18) providing a bunch of extra functionality.  While I'm unsure how much of this extra functionality is exercised by users of `ZeroInflated` distributions, I'm hesitant to demote these distributions without good reason.

Reply:
see my comment above